### DISCUSSION TITLE
My issues are hidden on other repositories.

### DISCUSSION THREAD

**Discussion by TorCzaf** [community/community#59565] [Projects and Issues]
### Select Topic Area

Question

### Body

Hi When I try to submit new issues on other repositories, I can't find my issues as unregistered user, even when I open the links of my issues on other browsers or as unregistered user I get 404 Didn’t find anything here!. 

I think my issues are completely  hidden from every user except me.
How do I fix this?
Thank You.

Edit: It is finally fixed. Thank You.

**Comment by ghost** [comment]
To troubleshoot the issue where your issues are hidden from every user except you, there are a few steps you can take:

1. Check the repository settings: Make sure that the repository you are submitting issues to does not have any specific settings that restrict visibility or access to issues. Check the repository's settings page on GitHub or the equivalent for other Git hosting platforms to ensure that issues are not restricted in any way.

2. Verify your user permissions: Ensure that you are logged in as a registered user with the appropriate permissions to view and submit issues. If you are logged in as an unregistered user or with limited permissions, you may not be able to see or access certain issues. Try logging in as a registered user with higher permissions to see if the issues become visible.

3. Check the issue visibility settings: In some cases, the issue visibility may be restricted to specific users or groups. Double-check the issue settings to ensure that the issues you are trying to access are not restricted to certain users or groups. You can refer to the documentation of the Git hosting platform you are using to understand how issue visibility settings work.

4. Verify the issue status: Make sure that the issues you are trying to access are not closed or resolved. Closed issues are often hidden by default, so you may not be able to see them unless you specifically search for closed issues or change the filter settings.

5. Review the repository code: If you have access to the repository's codebase, check for any custom code or plugins that may be hiding or restricting the visibility of issues. Look for any code that modifies the issue visibility or access permissions. For example, in the source code you provided, there is a mention of a plugin called "hide-old-shortcodes" that hides certain content. Verify if this plugin or any similar code is affecting the visibility of issues.

6. Seek support from the repository owner or platform support: If you have tried all the above steps and are still unable to resolve the issue, consider reaching out to the repository owner or the support team of the Git hosting platform you are using. They may be able to provide further assistance or insights into the issue.

Sources:
- [GitHub Issue #577](https://github.com/verdaccio/verdaccio/issues/577)
- [GitHub Issue #487](https://github.com/verdaccio/verdaccio/issues/487)
- [StackOverflow Answer](https://stackoverflow.com/questions/252698/is-it-normal-to-have-the-home-directory-under-git)

**Comment by TorCzaf** [comment]
Thank you I have already contacted support directly and they fixed it.

**Comment by alexkd11** [comment]
Sorry. I have the same problem as you. My issues are not visible to others. Tell me what and where corrections need to be made.
